taxi/form.py

```diff
@@ -0,0 +1,39 @@
+from django import forms
+from django.core.validators import RegexValidator
+
+from taxi.models import Driver, Car
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    license_number = forms.CharField(
+        required=True,
+        validators=[
+            RegexValidator(
+                regex=r"^[A-Z]{3}\d{5}$",
+                message="Number of licese must consist "
```

Comment:
There is a typo in the error message: 'Number of licese' should be 'Number of license'.